spec.bs

```diff
@@ -1345,7 +1350,9 @@ Each {{InterestGroupReportingScriptRunnerGlobalScope}} has a
        href=#evaluate-script-return>return</a></i>.
 
     1. Set |finalCompletion| be [=ECMAScript/Completion Record|Completion=]([$Call$](F, `undefined`,
-       |arguments|)).
+       |arguments|)). In |timeout| milliseconds, if the invocation of [$Call$] has not completed,

```

Comment:
Can you move "In [...]" to a new line below "Set" but in the same algorithm step?